AutoCADCommands/Test.cs

```diff
@@ -1003,12 +1003,20 @@ public void TestPolygonMesh()
         }
 
         [CommandMethod("TestAddAttribute")]
-        public void TestAddAttribute()
+        public void TestAppendAttribute()
         {
             var iId = Draw.Insert("test", Point3d.Origin);

```

Comment:
Let's not prefix variable names. `iId` -> `id` or `insertId`

Reply:
oops, good catch.